libvast/vast/concept/parseable/string/quoted_string.hpp

```diff
@@ -13,13 +13,14 @@
 
 #pragma once
 
-#include <cassert>
-#include <string>
-
 #include "vast/concept/parseable/core.hpp"
+#include "vast/concept/parseable/string/any.hpp"
```

Comment:
I don't see that used directly here. Leftover?

Reply:
Good catch. I removed the line.